comps/llms/text-generation/llamacpp/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM python:3.11-slim
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
+    curl \
+    libgl1-mesa-glx \
+    libjemalloc-dev
```

Comment:
What these are needed for?

Reply:
I believe the two libjemalloc-dev, libgl1-mesa-glx are required by packages in requirements.txt..  62 OPEA components use libjemalloc-dev, and 63 components use libgl1-mesa-glx

When I remove them, and I get an ASGI error, with trace including fastapi, and prometheus-fastapi-instrumentator. I can remove the `curl` though, seems to work without installing curl.

Reply:
OPEA apps seem to be working fine without them.  I'll try to find is there actual use for them.

Reply:
You're right on this, the ASGI error was something else.  I've now removed them from the Dockerfile.